PyPDF2/filters.py

```diff
@@ -32,8 +32,9 @@
 
 import math
 import struct
-from io import StringIO
+from io import StringIO,BytesIO
 from typing import Any, Dict, Optional, Tuple, Union
+from PIL import Image
```

Comment:
Pillow (pil) is an optional dependency and should be imported within the function

Reply:
understood